OneSignalSDK/onesignal/src/main/java/com/onesignal/ADMMessageHandler.java

```diff
@@ -43,6 +44,18 @@
    public static class Receiver extends ADMMessageReceiver {
       public Receiver() {
          super(ADMMessageHandler.class);
+         boolean ADMLatestAvailable = false;
+         try{
+            Class.forName( "com.amazon.device.messaging.ADMMessageHandlerJobBase" );
+            ADMLatestAvailable = true ;
+         }
+         catch (ClassNotFoundException e)
+         {
+            // Handle the exception.
```

Comment:
can we log this?

Reply:
Don't need to log the catch as it is expected to throw on older FireOS devices everytime.

Logging `ADMLatestAvailable` after words as DEBUG would be a good idea though.